### DISCUSSION TITLE
can we Access uvdesk created on localhost in another system with same network.

### DISCUSSION THREAD

**Discussion by Thulasi2023** [uvdesk/community-skeleton#761] [Q&A]
i was unable to access the uvdesk which was creted on the localhost. i was unable to acess the it another system with same network.

when i open the port with **ngrok** i can accesss only the phymyadmin page.
 we are unable to open the agent login link and clinent link  in same network.

**Comment by nicokr01** [comment]
Hi what did you install UVdesk on (which OS ?). Check your firewall settings on your device, maybe it blocks the HTTP / HTTPS port.
